flink-runtime/src/main/java/org/apache/flink/runtime/state/KeyedBackendSerializationProxy.java

```diff
@@ -98,8 +103,7 @@ public int getVersion() {
 
 	@Override
 	public int[] getCompatibleVersions() {
-		// we are compatible with version 3 (Flink 1.3.x) and version 1 & 2 (Flink 1.2.x)
-		return new int[] {VERSION, 3, 2, 1};
+		return new int[]{VERSION, 4, 3, 2, 1};
```

Comment:
nit: miss a ' ' between '[]' and '{'.

Reply:
Both styles are ok and used in Flink, so I will stick to this.